setup.py

```diff
@@ -135,7 +135,8 @@ def copy_extensions_to_source(self):
         extdir = ext_fullpath.parent.resolve()
         so_files = os.path.join(extdir, '*.so')
         pyd_files = os.path.join(extdir, '*.pyd')
-        dll_files = os.path.join(self.build_temp, 'bin', 'Release', '*.dll')
+        cfg = "Debug" if self.debug else "Release"
+        dll_files = os.path.join(self.build_temp, '_pywhispercpp', 'bin', cfg, "*.dll")
```

Comment:
Oh, I forgot the to add the extension folder when I was committing!
Looks great :+1: